### ISSUE TITLE
/etc/perseo-core.properties missed and perseo-fe url as variable for docker

### ISSUE THREAD

**Issue by AlvaroVega** [telefonicaid/perseo-core#70]
- This file is missed in perseo-core deploys: /etc/perseo-core.properties
```
# This file should be copied by deployment process
# into /etc/perseo-core.properties, with the appropiate permissions

# URL for invoking actions when a rule is fired
action.url = http://<PERSEO_FE_URL>/actions/do

# Time in milliseconds (long) to "expire" a "dangling" rule
rule.max_age= 60000
```
- The variable PERSEO_FE_URL should be a parameter for docker container

**Comment by AlvaroVega** [comment]
https://github.com/telefonicaid/perseo-core/pull/71

**Comment by fgalan** [comment]
Fixed by PR #73
